pykokkos/core/translators/bindings.py

```diff
@@ -380,17 +380,46 @@ def generate_kernel(
 
     return kernel
 
-def bind_wrappers(module: str, wrappers: List[str]) -> str:
+def bind_wrapper(variable: str, wrapper: str) -> str:
     """
-    Generate the binding code for all wrappers
+    Generate the binding code for a wrapper
+
+    :returns: the binding code
+    """
+
+    binding = f"{variable}.def(\"{wrapper}\", &{wrapper});"
+
+    return binding
+
+def bind_functor(
+    variable: str,
+    functor: str) -> List[str]:
```

Comment:
I'm nitpicking here but can you put all the arguments on the same line?

Reply:
done in [4fba93](https://github.com/kokkos/pykokkos/pull/102/commits/4fba9361ee783695c9dd9bda0cc503729f21c72d)